### PR TITLE
formatting: Reformat files from allowlist

### PR DESCRIPTION
**Pull request by ldalek-antmicro**
Verible formatter version: v0.0-1213-g9e5c085b

### CODE REVIEW

File: hw/ip/adc_ctrl/rtl/adc_ctrl.sv
Comment (ldalek-antmicro) [inline hw/ip/adc_ctrl/rtl/adc_ctrl.sv:55]: @sriyerg @udinator @msfschaffner @imphil
I'm not sure about how such statements should be formatted.
Closed comment I've found is here https://github.com/lowRISC/ibex/pull/1215/files/51b6f79cd76e8596d56b8519cace94bae80b0512#diff-52a56b9b533fbf6ca22fe732d0423fb6ff3aacc305436bccf1a0273cae1c5351

Comment (udinator) [reply hw/ip/adc_ctrl/rtl/adc_ctrl.sv:55]: I'd prefer this formatted version to be used - maybe @msfschaffner can comment as well since this is RTL code

Comment (msfschaffner) [reply hw/ip/adc_ctrl/rtl/adc_ctrl.sv:55]: This formatted version looks fine to me.
Breaking out over multiple lines should only be needed when the statement does not fit onto a single line anymore...

**Review by msfschaffner — COMMENTED** [review]
RTL LGTM, I let the DV folks approve if DV code looks good to them.

File: hw/ip/adc_ctrl/rtl/adc_ctrl.sv
Comment (sriyerg) [reply hw/ip/adc_ctrl/rtl/adc_ctrl.sv:55]: this lgtm. thanks!

**Review by sriyerg — APPROVED** [review]

**Review by weicaiyang — APPROVED** [review]
LGTM

**Review by udinator — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]

**Comment by asb** [comment]
Hi @ldalek-antmicro. Thank you for the contribution and of course your work on improving Verible.

We (myself, @dsandrs, @tomroberts-lowrisc, and others) had a discussion recently about formatting-only changes and the main conclusion was that due to concerns about churn, review overhead, and the risk of inadvertently introducing functional changes we should hold off on any such of mass reformatting effort for the time being.

This PR has had plenty of review so at this point I've got no particular objection to it landing, but beyond that I think improvements to our pre-commit testing are the best area to focus efforts (i.e. ensuring any incremental changes and newly added code are formatted according to our styleguide).

To that end, do I understand correctly that the in-tree verible formatting script currently doesn't support limiting reformatting to just the changed lines? From a quick look it seems verible does support --lines as clang-format does, so a similar approach to clang-format-diff could be used?

**Comment by msfschaffner** [comment]
CC @hzeller @tansell

**Comment by tgorochowik** [comment]
Hi @asb 

Thank you for your feedback! We understand your concerns, we will try to come up with some ideas on how we can proceed in a way that will allow us to avoid the overhead and the risks you mention.

Your suggestion about trying to reformat only the changed lines sounds great, we will experiment a little bit to see in what way our work can be the most useful for OpenTitan.

**Pull request merged**
